modules/ROOT/pages/sync-update-api-spec.adoc

```diff
@@ -1,23 +1,26 @@
-= Update the API Specification in Your Project
+= Update the API Spec in Your Project
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-When you import an API specification as a dependency, you can update the API specification version in your Studio project.
+When you import an API spec as a dependency, you can update the API spec version in your Studio project if there's a newer version available.
 
-To update the API specification:
+To update the API spec:
 
-. In *Package Explorer*, right-click the Mule project that contains the API specification to update.
+. In *Package Explorer*, right-click the Mule project that contains the API spec to update.
 . Select *API Specs* > the name of your API > *Update Version*.
 +
-After you update the specification, accept that Studio regenerates the flows based on the new version.
+[NOTE]
+The *Update Version* option is shown if there is a newer version of the API spec available.
 +
-To manually generate a flow for the updated API, right-click the Mule project where you updated the API specification, and select *API Specs* > the name of your API > *Generate Flows*.
+After you update the API spec, accept that Studio regenerates the flows based on the new version.
```

Comment:
I'd change the wording so that it's not that imperative. The user can avoid generation if by any change they don't need it. Maybe something in the line of:
`After you update the API spec, you will be asked if you want Studio to regenerate the flows based on the new version. Either accept or decline based on your needs.`

Reply:
OK! Thanks!